frontend/src/__tests__/pages/Admin/Contacts.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import adminRoutes from "../../../routes/subRoutes/adminRoutes";
+import { MemoryRouter, Routes } from "react-router-dom";
+import { Contacts } from "../../../pages/Admin/Contacts";
+import { shallow } from "enzyme";
+
+//Mock keycloak.
+jest.mock("@react-keycloak/web", () => ({
+  useKeycloak: () => ({ initialized: true, keycloak: { authenticated: true } }),
+}));
+
+describe("<Contacts /> routing", () => {
+  it("renders Contacts page when '/admin' is hit", () => {
```

Comment:
shouldn't it be /admin/contacts

Reply:
Test don't have naming conventions as far as I know.  Some people use lower case but I prefer to follow the same naming conventions for React.  React uses capitals to describe a component, and camelcase to describe actions, reducers, providers, hooks, etc..  This method is easy to follow as it would be the same naming conventions for your apps structure.

Reply:
Here's someone using this example 

https://levelup.gitconnected.com/2-simple-effective-react-file-naming-convention-tips-cce1022328a8

Reply:
I think you missed the point "renders Contacts page when '/admin' is hit, ... it is not '/admin', but '/admin/contacts'. This is the description of the test, so it should reflect the test it is doing